source/sync/app-builder/tiered-device-sync/index.txt

```diff
@@ -0,0 +1,379 @@
+:noprevnext:
+:orphan:
+
+.. meta::
+   :robots: noindex, nosnippet
+
+.. _tiered-device-sync-guide:
+
+========================
+Tiered Device Sync Guide
+========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
```

Comment:
I've noticed this trend in a lot of our docs, but from a structural and SEO standpoint, it's not great to have one heading (page title) immediately followed by another (Overview).

Can we remove the "Overview" heading here? I know this is an internal doc and it's less important. Maybe this is something for a Realm Lobby discussion?

Reply:
Sure, I don't have a strong opinion. I just copied this from an existing page. Probably a good thing to discuss in Lobby!